go/vt/schemadiff/errors.go

```diff
@@ -239,3 +239,13 @@ func (e *InvalidColumnInForeignKeyConstraintError) Error() string {
 	return fmt.Sprintf("invalid column %s referenced by foreign key constraint %s in table %s",
 		sqlescape.EscapeID(e.Column), sqlescape.EscapeID(e.Constraint), sqlescape.EscapeID(e.Table))
 }
+
+type ColumnRenamedError struct {
+	Table  string
+	Column string
+}
+
+func (e *ColumnRenamedError) Error() string {
+	return fmt.Sprintf("column %s renamed in table %s",
+		sqlescape.EscapeID(e.Column), sqlescape.EscapeID(e.Table))
+}
```

Comment:
@shlomi-noach Are these errors used anywhere?

Reply:
I raced you and won, there are removed